Dockerfile

```diff
@@ -1,16 +1,13 @@
 FROM docker:latest as buildx_image
 
 ARG DOCKER_CONFIG=/env_configs/.docker
-ARG BUILDX_VERSION=0.6.0
```

Comment:
I am going to add this arg back so that users can build any version if they want

Reply:
Done!

Reply:
Excellent, that last touch up indeed did the trick. Teamwork!